src/crawlers/puppeteer_crawler.js

```diff
@@ -20,7 +20,7 @@ import { BrowserPoolOptions } from 'browser-pool';
 /**
  * @callback PuppeteerHook
  * @param {{ page: Page, crawler: PuppeteerCrawler } & BrowserCrawlingContext & CrawlingContext} crawlingContext
- * @param {DirectNavigationOptions} gotoOptions
+ * @param {any} gotoOptions
```

Comment:
This needs to be properly typed as well.

Reply:
why we have `any` here? it breaks the link here https://sdk.apify.com/docs/next/typedefs/puppeteer-hook

Reply:
@B4nan https://github.com/apify/apify-js/pull/1241#issuecomment-992240977

Reply:
58d2836dae43a615a9972f849384edab39750972